angular2_components/1-base/pubspec.yaml

```diff
@@ -1,13 +1,18 @@
 name: components_codelab
 description: A Dart app that uses Angular 2
```

Comment:
The description should be updated (in this pubspec and others).
At least "Angular 2" --> "AngularDart".